ipatests/prci_definitions/nightly_master.yaml

```diff
@@ -699,3 +699,15 @@ jobs:
         template: *ci-master-f28
         timeout: 19600
         topology: *ipaserver
+
+  fedora-28/test_ntp_replacement_TestNTPreplacement:
+    requires: [fedora-28/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-28/build_url}'
+        test_suite: test_integration/test_ntp_replacement.py::TestNTPreplacement
+        template: *ci-master-f28
+        timeout: 7200
+        topology: master_1repl_1client 
```

Comment:
This should be topology: *master_1repl_1client

Reply:
fixed